pootle/apps/pootle_language/models.py

```diff
@@ -76,9 +78,14 @@ def pootle_path(self):
         return '/%s/' % self.code
 
     @property
+    def iso_fullname(self):
+        """ISO fullname for the language."""
+        return get_language_iso_fullname(self.code) or self.fullname
+
+    @property
     def name(self):
         """Localized fullname for the language."""
-        return tr_lang(self.fullname)
+        return tr_lang(self.iso_fullname)
```

Comment:
Correct me if I'm wrong.  But this looks like if your code is found in TTK you are going to get the `iso_fullname` regardless if you have a simpler `fullname`.  While this is great in another language, it is a serious problem in English.